instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -280,11 +280,17 @@ def __call__(self, environ, start_response):
             start_response: The WSGI start_response callable.
         """
 
-        token = context.attach(extract(environ, getter=wsgi_getter))
```

Comment:
We aren't using the helper function we used for other packages here?

Reply:
you mean _start_Internal_or_server_span() function?
No we cant as there is a missing argument in that function 'attributes' which is being used by start_span() function in wsgi framework.